Mathlib/Topology/Separation/CompletelyRegular.lean

```diff
@@ -88,6 +88,19 @@ instance NormalSpace.instCompletelyRegularSpace [NormalSpace X] [R0Space X] :
   have hgK : EqOn g 1 K := fun k hk => Subtype.ext (hfK hk)
   exact ⟨g, cg, hgx, hgK⟩
 
+lemma Topology.IsInducing.completelyRegularSpace
+    {Y : Type v} [TopologicalSpace Y] [CompletelyRegularSpace Y]
+    {f : X → Y} (hf : IsInducing f) : CompletelyRegularSpace X where
+  completely_regular x K hK hxK := by
+    rw [hf.isClosed_iff] at hK; obtain ⟨K, hK, rfl⟩ := hK; rw [mem_preimage] at hxK
```

Comment:
we generally only allow `;` if the entire tactic sequence to close the goal fits on that line.
```suggestion
    rw [hf.isClosed_iff] at hK
    obtain ⟨K, hK, rfl⟩ := hK
    rw [mem_preimage] at hxK
```